### PR TITLE
Use secure link to security policy

### PR DESCRIPTION
**Pull request by l0b0**
### Summary

Trivial URL fix.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #### **Did you find a bug?**
 
 * **Do not open up a GitHub issue if the bug is a security vulnerability
-  in Rails**, and instead to refer to our [security policy](http://rubyonrails.org/security/).
+  in Rails**, and instead to refer to our [security policy](https://rubyonrails.org/security/).
```
Comment (jeremy) [inline CONTRIBUTING.md]: rubyonrails.org is http-only currently.

Comment (l0b0) [reply CONTRIBUTING.md]: @jeremy Weird, I swear the HTTPS link worked a few hours ago.

Comment (jeremy) [reply CONTRIBUTING.md]: We're hosted with GitHub pages, so when/if they support https on apex domain, the link will work! Thanks for the PR in any case 😊

Comment (esparta) [reply CONTRIBUTING.md]: Don't know if this has been answered before, but... is there any plan (or has been rejected) to use Cloudflare as an intermediate to get the SSL done?

**Pull request closed without merging**